functions.lua

```diff
@@ -285,10 +282,10 @@ function telemosaic.rightclick(pos, node, player, itemstack, pointed_thing)
 
 	elseif item == "telemosaic:key" then
 		-- Try to set a new destination
-		local dest_hash = itemstack:get_meta():get_string("")
-		local src_hash = core.pos_to_string(pos)
-		if dest_hash ~= src_hash and not minetest.is_protected(pos, player_name) then
-			local dest = unhash_pos(dest_hash)
+		local dest = itemstack:get_meta():get_string("")
+		local src = core.pos_to_string(pos)
```

Comment:
how about ``dest_string`` and ``src_string`` as short for ``destination_position_string``.
Or, middle ground, ``destination_string`` and ``source_string``

Of course one could consider changing the signature variable to ``destination`` but it's probably nicer to change the 'new' ones.

Either way, that should satisfy luacheck.

Reply:
done in https://github.com/mt-mods/telemosaic/pull/27/commits/16c7969c16a58bcafad970fcb8affbcee8f3f324